AwesomePj.html

```diff
@@ -12,5 +12,6 @@
   너네가 일할곳
   <!-- 여기 밑에다가 코딩해!! -->
   
+  이제는 제대로 카운터가 잘 만들어졌겠죠??? 제발!
```

Comment:
```suggestion
  이제는 엄청난 카운터가 잘 만들어졌겠죠??? 제발!
```
기능이 어썸하지 않네요? 이런식으로 수정하면 좋을 것 같아요.